### PR TITLE
RELATED: ONE-1229 Skip execution of "all time" and "select all" filters

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/114/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/115/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/117/

File: src/execution.js [hunk 1]
```diff
@@ -303,6 +303,12 @@ const dateFilterToWhere = f => {
 };
 
 const metricToDefinition = metric => ({ element: get(metric, 'objectUri') });
+const isDateFilterExecutable = dateFilterSettings =>
+    get(dateFilterSettings, 'from') !== undefined &&
+    get(dateFilterSettings, 'to') !== undefined;
+
+const isAttributeFilterExecutable = listAttributeFilter =>
+    !isEmpty(get(listAttributeFilter, ['default', 'attributeElements']));
```
Comment (akloboucnik) [inline src/execution.js]: use `notEmpty` (declared on line 25)

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/gooddata-js-pull-request-builder/118/

**Pull request merged**